packages/ui/src/Table/features/columnsSelection/ContextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react'
```

Comment:
Should we add e2e tests for the new component?

Reply:
We should. I just don't know when we could setup it

Reply:
I talking about screenshot/loki tests, that we have already.

Reply:
done